utils/command_line_params.cpp

```diff
@@ -912,3 +914,9 @@ void add_su3_option_group(std::shared_ptr<QUDAApp> quda_app)
   opgroup->add_option("--su3-measurement-interval", measurement_interval,
                       "Measure the field energy and topological charge every Nth step (default 5) ");
 }
+
+void add_split_grid_option_group(std::shared_ptr<QUDAApp> quda_app)
```

Comment:
I like this suggestion here.  Would make things more logical.

Reply:
Shall we already rename this group such that it can be used for all communication options? Maybe call it `comms` @maddyscientist @hummingtree ? And file an issue to complete that cleanup?

Reply:
I have added this into #1091.